news/2017-08-17-scorewatch-august-weeks-1-and-2.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Scorewatch: August Weeks 1 & 2"
+date: 2017-08-17 07:30:00 +0000
+---
+
+Two weeks in one! Things are getting settled after many of osu!'s biggest names return from the YACA meetup in China, but there's still sweet scores to be had. Another player has joined the 700pp club, a special SS set on a difficult osu!mania map, and much more in this special fortnightly edition of Scorewatch!
+
+![](/wiki/shared/news/2017-08-17-scorewatch-august-weeks-1-and-2/banner.jpg)
+
+[_RyuK](https://osu.ppy.sh/u/6304246) has broken into the 700pp club! The number 1 Canadian set his mark on the famous map [kradness&Reol - Remote Control](https://osu.ppy.sh/b/774965?m=0) and taking an outstanding 97.76% HDDT 1 miss score to earn an amazing 704pp score! Well, I don’t think I need to write much more for this score. Have a look!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/b666KhdtrMU" frameborder="0" allowfullscreen></iframe></br>
+
+An unfamiliar face to newcomers, but definitely a player that has not rusted a bit! [Niko](https://osu.ppy.sh/u/175141) has come back strong with an amazing 98.59% S rank on [Tatsh - IMAGE -MATERIAL- <Version 0>](https://osu.ppy.sh/b/252238), taking the 7th only no mod S on this map, with an accuracy that even beat out [rrtyui](https://osu.ppy.sh/u/352328) himself! What a comeback from this amazing player!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/85Z1LMFMUK8" frameborder="0" allowfullscreen></iframe></br>
+
+[HappyStick](https://osu.ppy.sh/u/256802) has produced an incredible score on [Tech N9ne - Worldwide Choppers](https://osu.ppy.sh/b/345189?m=0), in fact, getting the first ever no mod S on the map and the No. 1 spot with a 98.79% 2225x combo play to earn 372pp. What a shame that an FC could have happened, but nonetheless, enjoy this livestream moment!
```

Comment:
would remove "in fact," and change last line to something like "while a shame the fc didn't happen, enjoy this livestream moment nonetheless"